vivisect/__init__.py

```diff
@@ -606,6 +611,9 @@ def analyze(self):
         Call this to ask any available analysis modules
         to do their thing...
         """
+        if self.getMeta('GUID') == None:
```

Comment:
sounds good to me.  we definitely *don't* want to be applying multiple GUIDs to a workspace... evar.  i'm writing this in.  PR soon.